Migration.md

```diff
@@ -558,6 +558,7 @@ ABI was broken for `sdf::Element`, and restored on version 11.2.1.
 ### Additions
 
 1. **world.sdf**: A joint can be specified directly in a world.
+1. **world.sdf**: Merge-includes are now allowed in worlds. The included models must not contain links.
```

Comment:
From https://github.com/azeey/sdformat/blob/world_merge_include/src/parser.cc#L374, it looks like `gripper` is not supported as well.

Reply:
Done in https://github.com/gazebosim/sdformat/pull/1233/commits/e1216672856481b1cfcce590546e2d1d8c676e8e